src/jsx/import/general/submit-button.jsx

```diff
@@ -7,25 +7,25 @@
  * Note: this script implements jsx (reactjs) syntax.
  */
 
-import React from 'react';
+import React, { Component } from 'react';
 
-var Submit = React.createClass({
-  // triggered when 'state properties' change
-    render: function(){
-        const buttonValue = !!this.props.btnValue ? this.props.btnValue : 'Submit';
-        const clickCallback = !!this.props.onClick ? this.props.onClick : '';
-        const cssClass = !!this.props.cssClass ? this.props.cssClass : 'form-submit';
+class Submit extends Component {
+    // triggered when 'state properties' change
+    render() {
+        const buttonValue = this.props.btnValue ? this.props.btnValue : 'Submit';
+        const clickCallback = this.props.onClick ? this.props.onClick : '';
+        const cssClass = this.props.cssClass ? this.props.cssClass : 'form-submit';
 
-        return(
+        return (
             <input
-                type='submit'
+                type="submit"
```

Comment:
Quick question, does the eslinter prefer double quotes over single quotes. I often prefer single quotes, in almost all cases, except when variables need to be interpolated, within the string. Granted, this preference is a generalization of mine across all languages.

Reply:
Is there anyway to have the linter prefer `type='submit'`, or is this general es6 rule?

Reply:
I'm pretty sure that there is, as this rule is just to the code be consistent where to use single or double quotes.
Not a es6 rule by any means

Reply:
Can we change double to single quotes?

Reply:
Sure!
You mean change the file in the test/lint folder?

Reply:
Sure!
Tomorrow I will finish this PR, if everything goes the right direction and will change back the quotes